### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by creiser** [NVlabs/EdgeRunner#8]
Sorry for opening another issue on this, but before you release the subset list that you used for training, could you maybe already give some hints on data.

I am currently using the subset of Objaverse that is described in https://raw.githubusercontent.com/ashawkey/objaverse_filter/refs/heads/main/kiuisobj_v1_merged_80K.csv
Does that file roughly correspond to filtering that you describe in the paper?

Best,
Christian

**Comment by ashawkey** [comment]
Hi, I haven't got the time to clean the list recently, but you may use [this](https://huggingface.co/ashawkey/misc/resolve/main/gob_all_117462_valid.json) list. You may still need to filter by number of faces.

**Comment by creiser** [comment]
Thank you. I am not sure how to use this list though. It does not seem to correspond to paths in the Objaverse dataset. Is this for Objaverse or Objaverse-XL or both?

Objaverse paths looks like this `objaverse_1_0/glbs/000-052/cfd1a14262c9452a98265ff51df738da.glb`, but the provided file contains entries like this `150/763878`.

**Comment by ashawkey** [comment]
Yes, it's using [Gobjaverse](https://github.com/modelscope/richdreamer/blob/main/dataset/gobjaverse/README.md) indices. Our DiT model is trained using Gobjaverse renderings, so I uses their indices. You can use this mapping to convert them back to Objaverse indices:
``` bash
wget https://virutalbuy-public.oss-cn-hangzhou.aliyuncs.com/share/aigc3d/gobjaverse_280k_index_to_objaverse.json
```

**Comment by creiser** [comment]
Thank you! Do you also have a similar list for Objaverse-XL?

**Comment by ashawkey** [comment]
Hi, our XL list is pretty dirty, maybe you could try [TRELLIS-500K](https://github.com/microsoft/TRELLIS/blob/main/DATASET.md).

**Comment by creiser** [comment]
Thank you. I have tried to train the ArAE with the provided [list](https://huggingface.co/ashawkey/misc/resolve/main/gob_all_117462_valid.json). I trained on 8 GPUs for 100 epochs with the provided config, but the reconstructed meshes look like complete garbage. Reconstructed meshes only have a few faces. I turned off mesh decimation augmentation because it lead to segmentation faults occacisonally. Could it be that mesh decimation augmentation is that important? Do you have any idea what might be the problem?

**Comment by clearfrosty** [comment]
Hello, it seems that you have successfully obtained the data set, could you please tell me how to download the data set corresponding to the path mentioned in the code? Thank you very much!
